app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -206,6 +207,64 @@ class QuestionPlayerActivityTest {
     }
   }
 
+  @Config(qualifiers = "xxhdpi")
```

Comment:
Out of curiosity, why isn't the default ``port-xxhdpi`` qualifier at the class-level sufficient? That should be the same configuration.

Reply:
Done